### PR TITLE
Add test scenarios and remediation for network ipv6 {disable rpc|privacy extensions} 

### LINKED ISSUE #2577: rhel 7.5 beta: network_ipv6_privacy_extensions still fails after remediation

### PR DESCRIPTION
**Pull request by yuumasato**
#### Description:

- Add test scenarios and bash remediation for `network_ipv6_disable_rpc` 
- Add test scenarios and bash remediation for `network_ipv6_privacy_extensions` 

#### Rationale:

- Cited rules are missing bash remediation
- Tests help ensure that remediations work

- Fixes #2577

### CODE REVIEW

File: shared/fixes/bash/network_ipv6_disable_rpc.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# platform = multi_platform_rhel
+
+# Drop 'tcp6' and 'udp6' entries from /etc/netconfig to prevent RPC
+# services for NFSv4 from attempting to start IPv6 network listeners
+declare -a IPV6_RPC_ENTRIES=("tcp6" "udp6")
+
+for rpc_entry in ${IPV6_RPC_ENTRIES[@]}
```
Comment (matejak) [inline shared/fixes/bash/network_ipv6_disable_rpc.sh]: It is a good habit to double quote @-expanded bash arrays `"${IPV6_RPC_ENTRIES[@]}"`. In this case, you don't need an array, a space-delimited string would be just fine.

File: shared/fixes/bash/network_ipv6_privacy_extensions.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# platform = multi_platform_rhel
+
+# enable randomness in ipv6 address generation
+for interface in $(ls /etc/sysconfig/network-scripts/ifcfg-*)
```
Comment (matejak) [inline shared/fixes/bash/network_ipv6_privacy_extensions.sh]: Just writing `/etc/sysconfig/network-scripts/ifcfg-*` is better than the `ls` stuff as it is simple and handles names with spaces correctly: http://mywiki.wooledge.org/ParsingLs

File: tests/data/group_system/group_network/group_network-ipv6/group_configuring_ipv6/rule_network_ipv6_privacy_extensions/ipv6_privacy_enabled.pass.sh [hunk 3]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+#
+# profiles = xccdf_org.ssgproject.content_profile_ospp-rhel7
+
+for file in $(ls /etc/sysconfig/network-scripts/ifcfg-*)
```
Comment (matejak) [inline tests/data/group_system/group_network/group_network-ipv6/group_configuring_ipv6/rule_network_ipv6_privacy_extensions/ipv6_privacy_enabled.pass.sh]: ditto - you can remove the ls stuff.

File: tests/data/group_system/group_network/group_network-ipv6/group_disabling_ipv6/rule_network_ipv6_disable_rpc/rpc_disabled.pass.sh [hunk 4]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+#
+# profiles = xccdf_org.ssgproject.content_profile_ospp-rhel7, xccdf_org.ssgproject.content_profile_ospp-rhel7
+# remediation = bash
+
+sed -i "/^tcp6[[:space:]]\+tpi\_.*inet6.*/d" /etc/netconfig
```
Comment (matejak) [inline tests/data/group_system/group_network/group_network-ipv6/group_disabling_ipv6/rule_network_ipv6_disable_rpc/rpc_disabled.pass.sh]: What is the escaped underscore for? I haven't been able to find anything about it.

Comment (yuumasato) [reply tests/data/group_system/group_network/group_network-ipv6/group_disabling_ipv6/rule_network_ipv6_disable_rpc/rpc_disabled.pass.sh]: I actually don't know. I sourced from the remediation script itself. Will try to look into it.

Comment (yuumasato) [reply tests/data/group_system/group_network/group_network-ipv6/group_disabling_ipv6/rule_network_ipv6_disable_rpc/rpc_disabled.pass.sh]: In fact there is no need to escape underscores, removed the backslash.

**Comment by yuumasato** [comment]
@matejak Thanks for your suggestions, I have made changes following them.

**Comment by matejak** [comment]
Great job, I see that you have made even another additional improvements.

**Pull request merged**